src/Book.java

```diff
@@ -0,0 +1,49 @@
+import java.util.Objects;
+public class Book {
+
+    private String title;
+    private Author author;
+    private int year;
+
+    public Book(String title, Author author,int year) {
+        this.title = title;
+        this.author=author;
+        this.year = year;
+    }
+    public String getTitle() {
+        return this.title;
+    }
+    public Author getAuthor() {
+        return this.author;
+    }
+    public int getYear() {
+        return this.year;
+    }
+    public void setYear(int year) {
+        this.year = year;
+    }
+    public String getAuthorFullName(){
+        return author.getName()+" "+ author.getSurname();
```

Comment:
дублирование кода, можно упростить:
```suggestion
        return author.toString();
```